images/kernel/upgrade-config.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+KERNEL_BUILDER_IMAGE=luxas/kernel-builder:gcc-7-2
+
+if [[ $# != 2 ]]; then
+    echo "Usage: $0 [FROM] [TO]"
+    exit 1
+fi
+
+FROM=$1
+TO=$2
+VERSION=$(echo ${TO} | rev | cut -d- -f1 | rev)
```

Comment:
Nice trick here! -- suggesting a comment:
```suggestion
VERSION="$(echo ${TO} | rev | cut -d- -f1 | rev)"  # Extracts the trailing hyphenated field -- this is dependent on naming the resulting file properly (ex: ./versioned/config-amd64-5.4.13)
```